src/main/java/ttt/PlayerFactory.java

```diff
@@ -5,38 +5,28 @@
 
 public class PlayerFactory {
 
-    public enum Marks {
-        X ("X"),
-        O ("O");
-
-        public final String mark;
-
-        Marks (String mark) {
-            this.mark = mark;
-        }
-    }
 
     public static List<Player> create(int type) {
         Player player1;
         Player player2;
         if (type == 1 || type == 2) {
-            player1 = new HumanPlayer(Marks.X.mark);
-            player2 = new HumanPlayer(Marks.O.mark);
+            player1 = new HumanPlayer(Marks.X);
+            player2 = new HumanPlayer(Marks.O);
             if (type == 2) {
                 return Arrays.asList(player2, player1);
             }
         } else if (type == 3){
-            player1 = new HumanPlayer(Marks.X.mark);
-            player2 = new ComputerPlayer(new RandomLocationGenerator(), Marks.O.mark);
+            player1 = new HumanPlayer(Marks.X);
```

Comment:
Not sure if it adds anything, but now player1 and player2 could be referred to as playerX and playerO? Maybe this would be more in line with the domain?